spec/requests/swagger_docs/v7/state_benefit_type_spec.rb

```diff
@@ -0,0 +1,22 @@
+require "swagger_helper"
+
+RSpec.describe "state_benefit_type", type: :request, swagger_doc: "v7/swagger.yaml" do
+  path "/state_benefit_type" do
+    get("list state_benefit_types") do
+      tags "Lookups"
+      produces "application/json"
+
+      response(200, "successful") do
```

Comment:
confused about this - state benefit type is not versioned

Reply:
@starswan I have to specify the `swagger_doc` to make it appear in both v6 and v7 API docs, otherwise it defaults to v6

Reply:
Maybe it should be separate somehow?